projects/aca-content/folder-rules/src/rule-details/edit-rule-dialog.ui-component.spec.ts

```diff
@@ -77,20 +81,20 @@ describe('EditRuleDialogSmartComponent', () => {
     });
 
     it('should show a "create" label in the title', () => {
-      const titleElement = fixture.debugElement.query(By.css('[data-automation-id="edit-rule-dialog-title"]')).nativeElement as HTMLDivElement;
+      const titleElement = unitTestingUtils.getByDataAutomationId('edit-rule-dialog-title').nativeElement as HTMLDivElement;
```

Comment:
this automation id is repeated multiple times so please make it reusable similarly like others selectors